solutions/telemetry/quarkus/consumer/src/main/resources/META-INF/resources/index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
```

Comment:
If we are not going to provide with a nice web interface, we can remove this file or create a generic one (thinking on displaying the course title or something like that) instead of using the one generated by Quarkus. The same comment applies to the other html files.

Reply:
I removed these files